### ISSUE TITLE
The latest available dotnet build (1.0.0.001876) for Ubuntu is installed incorrectly.

### ISSUE THREAD

**Issue by troydai** [dotnet/cli#1955]
## Steps to reproduce
1. On an Ubuntu 14.04 build
2. Run `./scripts/obtain/install.sh -c beta -d ~/.dotnet` (destination is overridden)
Output:
```
11:39 $ ./scripts/obtain/install.sh -c beta -d ~/.dotnet
dotnet_install: Preparing to install .NET Tools from 'beta' channel to '/home/troydai/.dotnet'
dotnet_install: Latest Version: 1.0.0.001876
dotnet_install: Local Version: <none>
dotnet_install: Downloading dotnet-ubuntu-x64.latest.tar.gz from https://dotnetcli.blob.core.windows.net/dotnet/beta/Binaries/Latest
######################################################################## 100.0%
dotnet_install: Extracting tarball
dotnet_install: Moving new CLI into install location and symlinking
dotnet_install: Linking /home/troydai/.dotnet/cli/dotnet -> /home/troydai/.dotnet/cli/bin/dotnet
dotnet_install: Cleaning dotnet-ubuntu-x64.latest.tar.gz
```

## Expected  behavior
The link `~/.dotnet/cli/dotnet` works.

## Actual behavior
The folder `~/.dotnet/cli/bin` doesn't exist therefore the link is not functional.

__Note__ on Mac the latest dotnet build from beta is __1.0.0.001884__. The ubuntu build is behind. Is it a problem in your build system or the build __1.0.0.001884__ for ubuntu is flawed therefore not published?

## Environment data
`dotnet --version` output:
_N/A_

The latest commit in the cli repo: `14d5dd80b07f9d49a5e1318e5fa7179b111bebfc`.

**Comment by troydai** [comment]
It occurs to me to be a deeper problem here. The build offered on different platforms are not aligned. Versions are different therefore the features are different. It makes consumers' lives difficult if their products are cross-platform. 

For example, OmniSharp builds on different platforms. The inconsistency between platforms makes their build systems complex.

**Comment by hal-ler** [comment]
Reverting `04adb5ef567b498fbb2ff5cdaf2b7a06fc521383` on install.sh seemed to have solved it locally.

**Comment by troydai** [comment]
Fixed in build `1.0.0.001897`
